packages/tfchain_client/src/client.ts

```diff
@@ -276,29 +276,38 @@ class Client extends QueryClient {
   }
 
   checkInputs(): void {
-    if (!this.url) throw new ValidationError("url should be provided");
+    if (!this.url) {
+      throw new ValidationError("URL is required.");
```

Comment:
please make the error msg more clear, what is the url required, i remember we saw this error before and we didn't get it right

Reply:
> please make the error msg more clear, what is the url required, i remember we saw this error before and we didn't get it right

Legit, Let's make it `TFChain URL is required`

Reply:
done